universal-application-tool-0.0.1/conf/evolutions/default/17.sql

```diff
@@ -0,0 +1,51 @@
+# --- Create the versions table and migrate existing actives and drafts.
+
+# --- !Ups
+create table versions (
+  id bigserial primary key,
+  lifecycle_stage varchar not null,
+  submit_time timestamp
+);
+
+create table versions_questions (
+  questions_id bigint,
```

Comment:
Since they're in the primary key, we'd be limited to only one null element, but it's not really too important since these are managed by ebeans ManyToMany annotations, so nulls aren't going to get inserted.  I'll add it since it doesn't matter much either way.